README.md

```diff
@@ -62,80 +75,124 @@ cloud.name=${CLOUD_NAME}
+- Java 11
+- Docker
+- Maven
+- PostgreSQL
+- IntelliJ IDEA (optional)
 
-### Clone
+### How to run local
+__Run with Docker__
+- use `docker-compose up` command `home-dev/init` 
+  in package to create  database
+- use `docker-compose up` command `home-dev/launch` in package to run 
+  application
 
-- Clone this repo to your local machine using `https://github.com/ita-social-projects/Home`
+__Run with Maven + Intellij IDEA__
+- use `mvn clean install` command in the root directory to build project
+- use `java -jar ./target/home-data-migration-0.0.1-SNAPSHOT.jar --url=jdbc:postgresql://localhost:5432/postgres -u=user -p=password`
+  command for connection to your local DB
+- run application with your IDEA
 
-### How to run local
-About it you can read in `hom-dev` package.
+If you did everything correctly, you should be able to access RapiDoc
+by this URL: http://localhost:8080/api/0/apidocs/index.html
+
+Detail info about local running or about other running options you can read in 
+[hom-dev](https://github.com/ita-social-projects/Home/tree/dev/home-dev) package.
 
 ---
 
-## Documentation
-- You can find OpenApi specification for the project [here](https://home-project-academy.herokuapp.com/api/0/apidocs/index.html)
+### Docker images
+Our [images](https://hub.docker.com/u/homeacademy) on Docker Hub:
+
+- ***data-migration*** - this image starts a data-migration for database in docker container
+
+- ***home-application*** - this image starts an application in docker container
 
 ---
 
-## Contributing
+## Documentation
 
-Our images on Docker Hub
+- You can find OpenApi specification for the
+  project [here](https://home-project-academy.herokuapp.com/api/0/apidocs/index.html).
 
-    https://hub.docker.com/u/homeacademy
+---
 
+## Contributing
 ### Git flow
 > To get started...
 #### Step 1
 
 - **Option 1**
-    - 🍴 Fork this repo!
+  - 🍴 Fork this repo!
 
 - **Option 2**
-    - 👯 Clone this repo to your local machine using `https://github.com/ita-social-projects/Home.git`
+  - 👯 Clone this repo to your local machine using command:
+
+`git clone https://github.com/ita-social-projects/Home.git`
 
 #### Step 2
 
-- **HACK AWAY!** 🔨🔨🔨
+Create your Feature Branch
+
+`git checkout -b name_for_new_branch`
 
 #### Step 3
+Make changes and test
 
-- 🔃 Create a new pull request using <a href="https://github.com/ita-social-projects/Home/compare/" target="_blank">github.com/ita-social-projects/Home</a>.
+#### Step 4
+Open a Pull Request with description of changes
 
 ---
 
 ## Team
-- Technical Expert of the project - @kh0ma
-- Mentor of SoftServe ITA - @DzigMS
+
+- Technical Expert of the project - `@kh0ma`
+- Mentor of SoftServe ITA - `@DzigMS`
 
 Developers that have worked on a project during:
-#### September - December 2020
-- @mykytam
-- @reeztl
-- @Prosperro-de
-- @alexorlenko
-- @omelchenkotimur
-- @Arthurpanda
-
-#### December 2020 - March 2021
-
-- @alexorlenko
-- @Dandria1802
-- @Romanov-Niko
-- @PoLytvy
-- @OlTym
-- @Abhai2016
+
```

Comment:
Can we just use the following markdown?
```
[![](https://opencollective.com/html-react-parser/contributors.svg?width=890&button=false)](https://github.com/ita-social-projects/Home/graphs/contributors)
```
[![](https://opencollective.com/html-react-parser/contributors.svg?width=890&button=false)](https://github.com/ita-social-projects/Home/graphs/contributors)

Or follow this article and make it more efficiently https://remarkablemark.org/blog/2019/10/17/github-contributors-readme/